app/actions.ts

```diff
@@ -0,0 +1,53 @@
+'use server';
+import prisma from '@/lib/prisma';
+
+const slugify = (text: string) =>
```

Comment:
I'm surprised they don't have a method for this already. Maybe NextJS implements one?